assets/js/modules/analytics/components/module/ModuleOverviewWidget/index.js

```diff
@@ -74,29 +76,46 @@ export default function ModuleOverviewWidget( { Widget, WidgetReportError } ) {
 	const statsReport = useSelect( ( select ) => select( MODULES_ANALYTICS ).getReport( statsArgs ) );
 	const statsError = useSelect( ( select ) => select( MODULES_ANALYTICS ).getErrorForSelector( 'getReport', [ statsArgs ] ) );
 
-	if ( overviewError || statsError ) {
-		return <WidgetReportError statsError={ overviewError || statsError } />;
+	const isZero = isZeroReport( overviewReport );
+	const isError = overviewError || statsError;
+
+	if ( ! overviewLoaded || ! statsLoaded ) {
+		return <ProgressBar />;
 	}
 
+	const reportsAreValid = ! isError && ! isZero;
+
 	return (
-		<Widget Header={ Header } noPadding>
-			<Overview
-				loaded={ overviewLoaded }
-				report={ overviewReport }
-				selectedStat={ selectedStat }
-				handleStatSelection={ setSelectedState }
-			/>
+		<Widget
+			Header={ Header }
+			noPadding={ reportsAreValid }
+		>
+			{ isError && (
+				<WidgetReportError moduleSlug="analytics" error={ overviewError || statsError } />
+			) }
+			{ ! isError && isZero && (
```

Comment:
Don't we need to wrap this to prevent some unexpected output from `isZero`?
```suggestion
			{ ( ! isError && isZero ) && (
```

Reply:
Updated.